### DISCUSSION TITLE
[Question][Hw3] Make command does not make .ll files

### DISCUSSION THREAD

**Discussion by ccloe4** [prosyslab-classroom/cs348-information-security#547] [Q&A]
Make command works but does not make .ll files.
All of below version is 10.0.0
clang --version
opt --version
llvm-config --version 

and I also did  > eval $(opam env)

Can you help me?
Thank you

**Comment by RiceBiscuits** [comment]
Where did you run 'make' command? 
.ll files are created only if you run 'make' under the test directory.

**Comment by ccloe4** [comment]
Oh I see, If I run 'make' under the root directory then it does not make .ll files under test directory. Am I right?

Thank you!

And I have another issue :(.. I used vscode to access the server and every time I enter the server, It shows error message like below figure.

![image](https://user-images.githubusercontent.com/79613844/117157216-5f2a4600-adf9-11eb-975d-6776b0ddddea.png)

![image](https://user-images.githubusercontent.com/79613844/117157254-681b1780-adf9-11eb-9f02-4017e26fd55a.png)

So the ocamlmerlin is not working.. Could you help me please?

**Comment by mickowale** [comment]
Try the following steps:
1. Remove OCaml and Reason IDE Extension
2. `opam install merlin` on the terminal
3. Install Ocaml and Reason IDE Extension
4. Reload if it says the same thing after step 3

Hope this works.

**Comment by ccloe4** [comment]
Thank you I'll try it

**Comment by ccloe4** [comment]
It doesn't work for me..
